### PR TITLE
unify "watch installation" section

### PR DESCRIPTION
**Pull request by evgeni**
1. prefer tmux over screen (screen is deprecated since EL7.6)
2. only put the Red Hat KCS link behind an ifdef
3. drop all unnecessary ifdefs

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.3/Katello 4.5
* [x] Foreman 3.2/Katello 4.4
* [x] Foreman 3.1/Katello 4.3
* For Foreman 3.0 or older, please create a separate PR.
* We do not accept PRs for Foreman 2.3 or older.

### CODE REVIEW

**Comment by evgeni** [comment]
RHEL 7.6 deprecation notes: https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html-single/7.6_release_notes/index#chap-Red_Hat_Enterprise_Linux-7.6_Release_Notes-Deprecated_Packages

File: guides/common/modules/proc_configuring-installation.adoc [hunk 1]
```diff
@@ -5,23 +5,12 @@ This initial configuration procedure creates an organization, location, user nam
 After the initial configuration, you can create additional organizations and locations if required.
 The initial configuration also installs PostgreSQL databases on the same server.
 
-ifdef::satellite[]
 The installation process can take tens of minutes to complete.
-If you are connecting remotely to the system, use a utility such as `screen` or `tmux` that allows suspending and reattaching a communication session so that you can check the installation progress in case you become disconnected from the remote system.
+If you are connecting remotely to the system, use a utility such as `tmux` or `screen` that allows suspending and reattaching a communication session so that you can check the installation progress in case you become disconnected from the remote system.
+ifdef::satellite[]
 For more information, see https://access.redhat.com/articles/5247[How to use the screen command] or alternately the `screen` manual page.
```
Comment (ekohl) [inline guides/common/modules/proc_configuring-installation.adoc:11]: If we only keep this in master I think we should drop this line. Master should be EL8+ and screen is gone.

**Review by ekohl — APPROVED** [review]
:+1: with one possible further improvement.

File: guides/common/modules/proc_configuring-installation.adoc (hunk 1 above)
Comment (evgeni) [reply guides/common/modules/proc_configuring-installation.adoc:11]: And if we merge this as is, and then do a follow up with the screen removal, we can pick the non-el8-only changes cleanly ;)

Comment (evgeni) [reply guides/common/modules/proc_configuring-installation.adoc:11]: (and there *is* screen in EPEL8 which people can decide to install)

File: guides/common/modules/proc_configuring-installation.adoc [hunk 2]
```diff
@@ -5,23 +5,12 @@ This initial configuration procedure creates an organization, location, user nam
...
-If you lose connection to the shell where the installation command is running, see the log at `{installer-log-file}` to determine if the process completed successfully.
-endif::[]
-
-ifdef::foreman-el,foreman-deb,katello[]
```
[the remark this answers is not in the document]
Comment (maximiliankolb) [reply guides/common/modules/proc_configuring-installation.adoc:15]: That's fine for me!

**Review by maximiliankolb — APPROVED** [review]

**Comment by evgeni** [comment]
To github.com:theforeman/foreman-documentation.git
   aa9c73ff2..2a4b6956c  3.1 -> 3.1
   f4e92d9bf..0ba980ff6  3.2 -> 3.2
   61ce5cef6..097590101  3.3 -> 3.3

**Pull request merged**